src/components/Navigator/NavigatorDataProvider.vue

```diff
@@ -31,7 +31,7 @@ export default {
      */
     technologyUrl: {
       type: String,
-      required: true,
+      default: '',
```

Comment:
I think you should probably leave this as required for now, since there isn't a realistic scenario (yet) where we would be providing data in this codebase without knowledge of a technology to fetch it for.